### PR TITLE
Adds FUSD payments to Storefront

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -51,6 +59,18 @@ curl --request POST \
 Run the commands below to mint new Kibble, create new items,
 and list some items for sale.
 
+- **POST /v1/FUSD/mint** - Mint new FUSD and send it to the `recipient` account.
```
Comment (psiemens) [inline api/README.md:62]:
```suggestion
- **POST /v1/fusd/mint** - Mint new FUSD and send it to the `recipient` account.
```

File: api/.env.local [hunk 2]
```diff
@@ -4,6 +4,7 @@ MINTER_ACCOUNT_KEY_INDEX=0
 
 FUNGIBLE_TOKEN_ADDRESS=${FUNGIBLE_TOKEN_ADDRESS}
 NON_FUNGIBLE_TOKEN_ADDRESS=${NON_FUNGIBLE_TOKEN_ADDRESS}
+FUSD_TOKEN_ADDRESS=${FUSD_ADDRESS}
```
Comment (psiemens) [inline api/.env.local:7]: I noticed that `FUSD_ADDRESS` in many other places -- does this need to be `FUSD_TOKEN_ADDRESS`?

File: web/src/hooks/use-fusd-balance.hook.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
 import {atomFamily, selectorFamily, useRecoilState} from "recoil"
-import {fetchKibblesBalance} from "../flow/script.get-kibbles-balance"
+import {fetchKibblesBalance} from "../flow/script.get-fusd-balance"
```
Comment (psiemens) [inline web/src/hooks/use-fusd-balance.hook.js:2]:
```suggestion
import {fetchFUSDBalance} from "../flow/script.get-fusd-balance"
```

File: web/src/flow/script.get-fusd-balance.js [hunk 4]
```diff
@@ -3,10 +3,10 @@ import {Address} from "@onflow/types"
...
   import FungibleToken from 0xFungibleToken
-  import Kibble from 0xKibble
+  import FUSD from 0xFUSD
 
   pub fun main(address: Address): UFix64? {
-    if let vault = getAccount(address).getCapability<&{FungibleToken.Balance}>(Kibble.BalancePublicPath).borrow() {
+    if let vault = getAccount(address).getCapability<&{FungibleToken.Balance}>(/public/fusdBalance).borrow() {
       return vault.balance
     }
     return nil
```
Comment (psiemens) [inline web/src/flow/script.get-fusd-balance.js:12]: should be `fetchFUSDBalance` or `getFUSDBalance` below

**Review by psiemens — COMMENTED** [review]
👏 

How does FUSD distribution work on testnet? The admin account will need FUSD minting permission on testnet in order for the `/fusd/mint` endpoint to work -- we can grant that permission while we wait for the FUSD faucet, if that makes sense.

File: api/.env.local (hunk 2 above)
Comment (10thfloor) [reply api/.env.local:7]: No this is fine. Prevents infinite loop when expanding env vars.

**Pull request merged**